src/components/Users/Users.jsx

```diff
@@ -89,10 +51,3 @@ const StyledPaginator = styled.div`
   display: flex;
```

Comment:
Bad practice to use margin instead of padding when you can use padding. You can try to investigate it.